serverless-plugin/default-config.yaml

```diff
@@ -32,12 +32,14 @@ alarms:
       ExtendedStatistic: p99
       Threshold: 5000
   States: # Step Functions
-    Statistic: Sum
     ExecutionThrottled:
+      Statistic: Sum
```

Comment:
Keen to understand this part of the change.

Reply:
Will revert this one too